frontend/src/components/public_group/AmountPicker.js

```diff
@@ -44,7 +44,7 @@ export default class AmountPicker extends React.Component {
         )}
 
         <div className='max-width-1 mx-auto' style={{position: 'absolute', left: '0', right: '0', margin: '-25px auto'}}>
-          {!hasStripe && (
+          {(!hasStripe || (hasPaypal && hasStripe)) && (
```

Comment:
if both `hasStripe` and `hasPaypal` are true, then it shouldn't show the `Pay With PayPal` button.